torch/ao/quantization/fx/_model_report/_detector.py

```diff
@@ -0,0 +1,143 @@
+from typing import Any, Dict, Set, Tuple
+
+import torch
+import torch.nn as nn
+from torch.ao.quantization.fake_quantize import FakeQuantize
+from torch.ao.quantization.observer import ObserverBase
+from torch.ao.quantization.qconfig import QConfig
+from torch.fx import GraphModule
+from torch.nn.qat.modules.conv import _ConvNd as QatConvNd
+from torch.nn.qat.modules.linear import Linear as QatLinear
+
+# Default map for representing supported per channel quantization modules for different backends
+DEFAULT_BACKEND_PER_CHANNEL_SUPPORTED_MODULES: Dict[str, Set[Any]] = {
+    "fbgemm": set([nn.Linear, nn.Conv1d, nn.Conv2d, nn.Conv3d, QatLinear, QatConvNd]),
+    "qnnpack": set([nn.Linear, nn.Conv1d, nn.Conv2d, nn.Conv3d, QatLinear, QatConvNd]),
+    "onednn": set([nn.Linear, nn.Conv1d, nn.Conv2d, nn.Conv3d, QatLinear, QatConvNd]),
+}
+
+
+def _detect_per_channel(model: GraphModule) -> Tuple[str, Dict[str, Any]]:
```

Comment:
nit: this can be a normal module, should still work since we aren't using any fx specific methods